spreadsheet/formula/fnstatistical.go

```diff
@@ -121,6 +122,37 @@ func CountBlank(args []Result) Result {
 	return MakeNumberResult(count(args, countEmpty))
 }
 
+// CountIf implements the COUNTIF function.
+func CountIf(args []Result) Result {
+	// COUNT and COUNTA don't require arguments, COUNTBLANK does
```

Comment:
Is this used for COUNT/COUNTA/COUNTBLANK too? Not just COUNTIF?

Reply:
removed, this comment was occasionally copypasted